src/django/sqls/drop_calc_column_func.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS calc_column(numeric, numeric);
```

Comment:
There is no such function. Please use the actual types while deleting this function.